Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj

```diff
@@ -186,6 +188,7 @@
 				<Property Name="Bld_localDestDirType" Type="Str">relativeToProject</Property>
 				<Property Name="Bld_modifyLibraryFile" Type="Bool">true</Property>
 				<Property Name="Bld_previewCacheID" Type="Str">{34E58981-12FC-4B65-80DF-C32EC0A02763}</Property>
+				<Property Name="Bld_version.build" Type="Int">1</Property>
 				<Property Name="Bld_version.major" Type="Int">1</Property>
 				<Property Name="Destination[0].destName" Type="Str">Keysight 34401A DMM Measurement UI.lvlibp</Property>
 				<Property Name="Destination[0].path" Type="Path">../NI_AB_PROJECTNAME/BuiltUI/Keysight 34401A DMM Measurement UI.lvlibp</Property>
```

Comment:
Did you test loading of the UI with this project? It looks like the build output path for the ppl hasn't changed. I think that means the relative path will be broken at run time.